sci-libs/blis/blis-0.8.1.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 2019-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..10} )
+inherit python-any-r1
+
+DESCRIPTION="BLAS-like Library Instantiation Software Framework"
+HOMEPAGE="https://github.com/flame/blis"
+SRC_URI="https://github.com/flame/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="doc eselect-ldso openmp pthread serial static-libs 64bit-index"
+REQUIRED_USE="
+	?? ( openmp pthread serial )
+	?? ( eselect-ldso 64bit-index )"
+
+RDEPEND="
+	eselect-ldso? (
+		!app-eselect/eselect-cblas
+		>=app-eselect/eselect-blas-0.2
+	)"
+
+BDEPEND="${PYTHON_DEPS}"
+DEPEND="${RDEPEND}
+	${PYTHON_DEPS}
```

Comment:
Does blis have a Python linking or header dependency?

I think it only goes to `BDEPEND`.

Reply:
> Does blis have a Python linking or header dependency?

No. Blis uses Python for [flattening its headers](https://github.com/flame/blis/blob/b727645eb7a8df39dee74068f734da66322fe0b3/common.mk#L502).

> I think it only goes to `BDEPEND`.

I agree with it.